.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+name: lint
```

Comment:
```suggestion
name: Lint
```

Reply:
Just to match the other actions

Reply:
Ok, gonna fix this.

Also, I think this is gonna change any way. I have plans to run pylint and pytest in the same workflow, to reduce setup/teardown, work with a large python matrix, and only test python apps. But the pytest section is coming with the nose => pytest migration for analyzer.